### ISSUE TITLE
associate start and end labels with the relevant scale value

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-confidenceSlider#82]
Issue raised by external auditor and includes the following:

**Current behaviour:**
The scale labels “Confident” and “Not confident” (`labelStart`  and `labelEnd`) are read out as a single paragraph as the user moves through the page text and are not associated with the numbers on the scale. They are also coded incorrectly, using a mix of the `<label>` tag and `aria-label` attributes.

**Expected behaviour:**
The labels should refer to their related values, so a user navigating to these would hear e.g. “Confident, 1” and “Not confident, 10”. Users should be able to exit the slider control with the up arrow to hear these additional labels without needing to know advanced screen reader commands.

**Recommended fix**
Remove the `<label>` tags and `aria-label` attributes from the slider labels (“incomplete”/”complete”) and include visually-hidden text labels alongside the visible text to show the rating that relates to each label.

**Comment by kirsty-hames** [comment]
In regard to the scale labels, there's an open PR for Slider that addresses this issue https://github.com/adaptlearning/adapt-contrib-slider/pull/176. When approved, the same implementation should be applied here.

**Comment by kirsty-hames** [comment]
In regard to...
> Users should be able to exit the slider control with the up arrow to hear these additional labels without needing to know advanced screen reader commands.

...it isn't possible to use the up arrow to exit the slider as the up arrow key is used to increase/decrease the scale value. I believe this is expected behaviour when interacting with a form element.

**Comment by oliverfoster** [comment]
Judging from this rather old article https://groups.google.com/g/free-aria/c/rLyWGY1IZpA
I would agree that its implemented differently across screen readers depending on the settings. With the jsx version, we're not doing anything special, so the behaviour comes as is. Just check that the rangeslider version behaves the same.

**Comment by oliverfoster** [comment]
Can we close this?

Ref: https://github.com/adaptlearning/adapt_framework/issues/3357
